doc/manual/import.rst

```diff
@@ -360,11 +360,11 @@ Putting all API imports together
 
 If you prepare your contest configuration as we described in the previous
 subsections, you can also use the script that we provide in
-`misc-tools/import-contest.sh`.
+`misc-tools/import-contest.py`.
```

Comment:
What's with the extensions. Why do we do that?

Reply:
I just 'copied' it from the old one. I'm fine dropping it, just like our submit client also doesn't have it. What do the others think?

Reply:
I vote in favor of dropping